### PR TITLE
[docs] Introduce basic example

### PR DESCRIPTION
**Pull request by inikulin**
This example demonstrates some basic TestCafe workflows and can be referenced e.g. in getting started as a working example of tests. Also, it can be used by various recipes: for debugging demonstration and page model.
\cc @MargaritaLoseva @VasilyStrelyaev @AlexanderMoskovkin 

BTW, we should add smoke test for examples, what do you think @AlexanderMoskovkin ?

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
> we should add smoke test for examples, what do you think @AlexanderMoskovkin ?

Yes, we should. I'll do this

File: examples/basic/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "url": "git+https://github.com/DevExpress/testcafe.git"
+  },
+  "author": "Developer Express Inc. (https://devexpress.com)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/DevExpress/testcafe/issues"
+  },
+  "homepage": "https://github.com/DevExpress/testcafe#readme",
+  "devDependencies": {
+    "testcafe": "*"
```
Comment (inikulin) [inline examples/basic/package.json]: Need to add `chai` to deps

**Comment by inikulin** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
> \r-

You are reviewing your own PR? Nice...

**Comment by VasilyStrelyaev** [comment]
rebase plsss

**Comment by inikulin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
As I understand the page for this tests is not published yet, right?

**Comment by inikulin** [comment]
@AlexanderMoskovkin Yeah

**Comment by AlexanderMoskovkin** [comment]
So, we can run this tests as smoke tests once this page is availabe

File: examples/basic/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Basic TestCafe API example
+
+*A set of examples that illustrate how to use TestCafe API.*
```
Comment (VasilyStrelyaev) [inline examples/basic/README.md]:
```
*A set of sample tests that illustrate how to use TestCafe API.*
```

File: examples/basic/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## Running the example
+
+Clone repository to your machine and install dependencies:
```
Comment (VasilyStrelyaev) [inline examples/basic/README.md]:
```
Clone the repository to your machine and install the dependencies:
```

File: examples/basic/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+```sh
+git clone https://github.com/DevExpress/testcafe.git
+cd examples/basic
+npm install
+```
+
+Then just run `test` command:
```
Comment (VasilyStrelyaev) [inline examples/basic/README.md]:
```
Then, run the `test` command:
```

File: examples/basic/test.js [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+
+    // Let's submit our form
+    await t
+        .wait(500)
+        .click(page.submitButton);
+
+    const results = await page.results();
+
+    expect(results.innerText).contains('Bruce Wayne');
+});
```
Comment (VasilyStrelyaev) [inline examples/basic/test.js]: You probably mean `pick` where you write `peek` in this file?

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by inikulin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**